src/sources/mikroorm/config.ts

```diff
@@ -0,0 +1,29 @@
+import { MikroORM } from "@mikro-orm/core";
+import {Owner, Car, Seller} from "./models";
+import {Options} from "@mikro-orm/core/utils";
+
+const config: Options = {
+  entities: [
+    Owner,
+    Car,
+    Seller
+  ],
+  dbName: 'adminjs-mikroorm',
+  type: 'postgresql',
+  clientUrl: 'http://localhost',
+  port: 5437,
```

Comment:
can we use envs for this config?

Reply:
we will probably use this repo to host a new demo so we need to have db connections configurable